File: CS/ScrollBarAppearance/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...

namespace ScrollBarAppearance {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            XYDiagram diagram = (XYDiagram)chartControl1.Diagram;
            ScrollBarOptions scrollBarOptions = diagram.DefaultPane.ScrollBarOptions;

            scrollBarOptions.BackColor = Color.White;
            scrollBarOptions.BarColor = Color.Blue;
            scrollBarOptions.BorderColor = Color.Navy;
            scrollBarOptions.BarThickness = 15;
            scrollBarOptions.XAxisScrollBarAlignment = ScrollBarAlignment.Far;
            scrollBarOptions.XAxisScrollBarVisible = true;
        }
    }
}

File: VB/ScrollBarAppearance/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...

Namespace ScrollBarAppearance
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim diagram As XYDiagram = CType(chartControl1.Diagram, XYDiagram)
			diagram.EnableAxisXScrolling = True
			diagram.EnableAxisYScrolling = True
			diagram.EnableAxisXZooming = True
			diagram.EnableAxisYZooming = True
			Dim scrollBarOptions As ScrollBarOptions = diagram.DefaultPane.ScrollBarOptions
			scrollBarOptions.BackColor = Color.White
			scrollBarOptions.BarColor = Color.Blue
			scrollBarOptions.BorderColor = Color.Navy
			scrollBarOptions.BarThickness = 15
			scrollBarOptions.XAxisScrollBarAlignment = ScrollBarAlignment.Far
			scrollBarOptions.XAxisScrollBarVisible = True
		End Sub
	End Class
End Namespace